Constitution.txt

```diff
@@ -1,490 +1,264 @@
+
+4: The Trial of all Crimes, except in Cases of Impeachment, shall be by Jury; and such Trial shall be held in the State where the said Crimes shall have been committed; but when not committed within any State, the Trial shall be at such Place or Places as the Congress may by Law have directed.
 
 Section 3
-1: Treason against the United States, shall consist only in levying War against them, or in adhering to their Enemies, giving them Aid and Comfort. No Person shall be convicted of Treason unless on the Testimony of two Witnesses to the same overt Act, or on Confession in open Court.
+1: Treason against these United States, shall consist only in levying War against them, or in adhering to their Enemies, giving them material Aid and Comfort. No Person shall be convicted of Treason unless on the Testimony of two Witnesses to the same overt Act, or on Confession in open Court. In the event that a State should secede from the Union, no trial of Treason shall be brought, unless the seceding State shall have declared War against these United States.
 
 2: The Congress shall have Power to declare the Punishment of Treason, but no Attainder of Treason shall work Corruption of Blood, or Forfeiture except during the Life of the Person attainted.
 
 Article IV (Article 4 - States' Relations)
 Section 1
-Full Faith and Credit shall be given in each State to the public Acts, Records, and judicial Proceedings of every other State. And the Congress may by general Laws prescribe the Manner in which such Acts, Records and Proceedings shall be proved, and the Effect thereof.
+1: Full Faith and Credit shall be given in each State to the public Acts, Records, and judicial Proceedings of every other State. And the Congress may by general Laws prescribe the Manner in which such Acts, Records and Proceedings shall be proved, and the Effect thereof.
+
+2: The Government of each State shall have the right to challenge the Government of any other state to prove its Acts, Records, and Proceedings, and their compliance with this Constitution and with the Constitution of the state. When such a challenge is issued, the supreme Court shall have original jurisdiction.
 
 Section 2
 1: The Citizens of each State shall be entitled to all Privileges and Immunities of Citizens in the several States.
 
 2: A Person charged in any State with Treason, Felony, or other Crime, who shall flee from Justice, and be found in another State, shall on Demand of the executive Authority of the State from which he fled, be delivered up, to be removed to the State having Jurisdiction of the Crime.
 
-3: No Person held to Service or Labour in one State, under the Laws thereof, escaping into another, shall, in Consequence of any Law or Regulation therein, be discharged from such Service or Labour, but shall be delivered up on Claim of the Party to whom such Service or Labour may be due.11
```

Comment:
Here is one more relic of slavery, which caused many of the tensions leading up to the Civil War, that we are well rid of.